README.md

```diff
@@ -37,28 +42,28 @@ This will be the file that holds your S3 and SQS configurations.
       }
     }
     ```
-1. [Register or create a new GitHub organization](https://github.com/settings/organizations). Find your organization's ID by visiting `https://api.github.com/orgs/<your-org-name>` and copying the `id` into the whitelist of `organizations` in `config/local.js`.
+1. [Register or create a new GitHub organization](https://github.com/settings/organizations) with a name of your choosing. Then find your organization's ID by visiting `https://api.github.com/orgs/<your-org-name>` and copying the `id` into the whitelist of `organizations` in `config/local.js`.
     ```js
     organizations: [
       99999999 // your org added here
     ]
     ```
     The organization will need to grant access to federalist, which can be done during:
       * a first-time login with your GitHub credentials
-      * in the Settings -> Applications -> federalist view in your GitHub Account
-1. Type `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` in terminal.
-1. Visit https://login.fr.cloud.gov/passcode to get a one time passcode.
-1. Enter your passcode back into terminal.
+      * in the [Authorized OAuth Apps](https://github.com/settings/applications) tab in your GitHub Account settings
+1. Paste `cf login --sso -a https://api.fr.cloud.gov -o gsa-18f-federalist -s staging` into your terminal window.
+1. Visit https://login.fr.cloud.gov/passcode to get a Temporary Authentication Code.
+1. Paste this code into the terminal, and hit the return key. (For security purposes, the code won't be rendered in the terminal.)
 1. Type `cf apps`.
```

Comment:
This step can actually be removed, it doesn't do anything necessary for setup actually.

Reply:
awesome!

Reply:
Which lines specifically @jseppi? Feel free to remove.

Reply:
The line that reads: `1. Type `cf apps`.